test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryStoreTest.cs

```diff
@@ -23,5 +23,31 @@ protected override void AddRoleStore(IServiceCollection services, object context
         {
             services.AddSingleton<IRoleStore<IdentityRole>, InMemoryRoleStore<IdentityRole>>();
         }
+
+        protected override InMemoryUser CreateTestUser(string username = "", string email = "",
```

Comment:
I would merge the new pocos with TestUser and keep the TestUser name as opposed to the confusing IdentityUser name